deprecated/admin/taxonomy/class-taxonomy-content-fields.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * This class parses all the values for the general tab in the Yoast SEO settings metabox.
+ */
+class WPSEO_Taxonomy_Content_Fields extends WPSEO_Taxonomy_Fields {
+
+	/**
+	 * Returns array with the fields for the general tab.
+	 *
+	 * @deprecated 14.9
+	 * @return array
+	 */
+	public function get() {
```

Comment:
To ensure backwards compatibility and avoid accidentally breaking something in an addon, we could temporarily call `WPSEO_Taxonomy_Fields::get('content')`.